### ISSUE TITLE
Peer-To-Peer-Code-Review-Portfolio-Desktop-Version

### ISSUE THREAD

**Issue by sulaimonaa** [sheldonwafula/The-portfolio#9]
#Code is working perfectly but I do have some suggestions for you, Please check the suggestions below:

## In the HTML5 structure, I will suggest you make the indentation to be properly aligned

**Comment by Aqurds** [comment]
I agree with the above suggestions to make he code visually more easy to read and follow.
